spark-doris-connector/src/main/java/org/apache/doris/spark/load/DorisStreamLoad.java

```diff
@@ -505,4 +530,19 @@ private void handleStreamPassThrough() {
 
     }
 
+    /**
+     * compress data by gz compression algorithm
+     */
+    private byte[] compressByGZ(String content) throws IOException{
+        byte[] compressedData;
+        try(ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            GZIPOutputStream gzipOutputStream = new GZIPOutputStream(baos);
+        ){
+            gzipOutputStream.write(content.getBytes("UTF-8"));
+            gzipOutputStream.finish();
+            compressedData = baos.toByteArray();
+        }
+        return compressedData;
```

Comment:
This method recommends adding a unit test

Reply:
have submitted a unit test for the method